docs/examples/viz_helical_motion.py

```diff
@@ -68,94 +71,130 @@
 directions = np.array([[1, 0, 0]])
 heights = np.array([8])
 arrow_actor = actor.arrow(centers, directions, color_arrow, heights,
-                          resolution=20, tip_length=0.06, tip_radius=0.012,
+                          resolution=50, tip_length=0.06, tip_radius=0.012,
                           shaft_radius=0.005)
+arrow_actor.GetProperty().SetAmbient(1)
```

Comment:
Same here @SunTzunami. Please make a function in ``material.py`` that takes an input an actor and changes its ambience or use directly the function ``manifest_standard`` from the same file to chose the options that you need.

Reply:
Used manifest_standard. Done.